temba/msgs/models.py

```diff
@@ -831,9 +835,20 @@ def process_message(cls, msg):
 
         cls.mark_handled(msg)
 
+        # Chatbase parameters to track logs
+        chatbase_not_handled = True
+
         # if this is an inbox message, increment our unread inbox count
         if msg.msg_type == INBOX:
             msg.org.increment_unread_msg_count(UNREAD_INBOX_MSGS)
+        elif msg.msg_type == FLOW:
+            chatbase_not_handled = False
+
+        # Registering data to send to Chatbase API later
+        org = msg.org
+        if org.is_connected_to_chatbase():
```

Comment:
`org.is_connected_to_chatbase` parses the org config so if we're doing that, we might as well take the chatbase credentials from the org config, and skip calling `msg.as_task_json()` altogether, no?